app/models/voyager_lookup.rb

```diff
@@ -97,6 +101,40 @@ def context_based_status(loc, status)
       end
     end
 
+    def status_label(item)
+      if item[:patron_group_charged] == "CDL"
+        "Reserved for digital lending"
+      elsif status_includes?(available_statuses, item[:status])
+        "Available"
+      elsif status_includes?(long_overdue_statuses, item[:status])
+        "Long overdue"
+      elsif status_includes?(lost_statuses, item[:status])
+        "Lost"
+      elsif status_includes?(returned_statuses, item[:status])
+        "Returned"
+      elsif status_includes?(transit_statuses, item[:status])
+        "In transit"
+      elsif status_includes?(in_process_statuses, item[:status])
+        "In process"
+      elsif status_includes?(checked_out_statuses, item[:status])
+        "Checked out"
+      elsif status_includes?(missing_statuses, item[:status])
+        "Missing"
+      elsif status_starts?('on-site - ', item[:status])
+        "See front desk"
+      elsif status_starts?('on-site', item[:status])
+        "On-site access"
+      elsif status_starts?('pending order', item[:status])
+        "Pending order"
+      elsif status_starts?('order received', item[:status])
+        "Order received"
+      elsif status_starts?('on-order', item[:status])
+        "On-order"
+      else
+        item[:status]
+      end
+    end
```

Comment:
This is great. You pulled the info from the availability.es6 in orangelight, correct?

Reply:
Yes I just worked my way down the list in the orangelight javascript.